param/__init__.py

```diff
@@ -102,6 +103,27 @@ def named_objs(objlist):
     return objs
 
 
+def param_union(*parameterizeds, **kwargs):
+    """
+    Given a set of Parameterized objects, returns a dictionary
+    with the union of all param name,value pairs across them.
+    If warn is True (default), warns if the same parameter has
+    been given multiple values; otherwise uses the last value
+    """
```

Comment:
Maybe: "If the same parameter is given multiple times, uses the last value (warns by default in this case; pass warn=False to overwrite silently)." ?